packages/env/lib/download-source.js

```diff
@@ -26,11 +28,20 @@ module.exports = async function downloadSource( source, options ) {
  * Clones the git repository at `source.url` into `source.path`. If the
  * repository already exists, it is updated instead.
  *
- * @param {Source} source The source to download.
- * @param {Object} options
+ * @param {Source}   source             The source to download.
+ * @param {Object}   options
  * @param {Function} options.onProgress A function called with download progress. Will be invoked with one argument: a number that ranges from 0 to 1 which indicates current download progress for this source.
+ * @param {Object}   options.spinner    A CLI spinner which indicates progress.
+ * @param {boolean}  options.debug      True if debug mode is enabled.
  */
-async function downloadGitSource( source, { onProgress } ) {
+async function downloadGitSource( source, { onProgress, spinner, debug } ) {
+	debug = debug
```

Comment:
We're using `debug` for two different types of value here which could cause confusion. Maybe call the function `log` and the boolean `debug`?

Reply:
Sure